crates/core/src/db/schema.rs

```diff
@@ -8,7 +8,6 @@ table! {
         value -> Nullable<Text>,
         trait_type -> Nullable<Text>,
         id -> Uuid,
-        first_verified_creator -> Nullable<Varchar>,
```

Comment:
Losing first_verified_creator? don't think we want to do that.

Reply:
I forgot to add it as it was a separate migration